.azure/pipelines/jobs/demo.yml

```diff
@@ -0,0 +1,22 @@
+parameters:
+  name: ''
+  vmImage: ''
+
+jobs:
+  - job: ${{ parameters.name }}
+    displayName: '${{ parameters.name }} build and test'
+    dependsOn:
+      - linux
```

Comment:
Does not need to depend an all other platforms. I.e. `mac_demo` just needs to depend on `mac`.